app/components/common/BackButton.tsx

```diff
@@ -6,17 +6,18 @@ type Props = {
   action: (e?: React.MouseEvent) => void;
   width?: number;
   height?: number;
+  left?: number;
```

Comment:
I don't like the idea of a generic "left" prop.
The perfect solution is to not change anything in the BackButton component and just put it in the right place (aka in the right column).

If it is not possible or will require a lot of refactoring — then ok, let's tweak BackButton but rename prop to less generic stuff.
Mb something like "isUnderLeftColumn: boolean" :)

p.s.
Also, I'm not sure that we really need `width` and `height` props as well ;)